modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java

```diff
@@ -359,6 +370,7 @@ public HttpResponse sendLoginPost(HttpClient client, String sessionDataKey) thro
         return response;
     }
 
+
```

Comment:
remove unnecessary new line

Reply:
Fixed https://github.com/wso2/product-is/pull/8500/commits/f3c8c97719e2480642ca3e50a14c54fbb87ab2ce